api/backend_test.go

```diff
@@ -1290,3 +1293,68 @@ func TestChangeDatabasePassword(t *testing.T) {
 	require.NotNil(t, key)
 	require.Equal(t, acc.Address, key.Address)
 }
+
+func TestCreateWallet(t *testing.T) {
+	utils.Init()
+	password := "some-password"
+	tmpdir := t.TempDir()
+
+	b := NewGethStatusBackend()
+	createAccountRequest := &requests.CreateAccount{
+		DisplayName:           "some-display-name",
+		CustomizationColor:    "#ffffff",
```

Comment:
question that's beyond this pr.
In mobile we have a set of keywords for colors. Does this matter? perhaps we have to align desktop & mobile for these? 🤔